src/QirRuntime/test/Standalone/InputTypes/generate.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import sys, os, platform, subprocess, datetime, shutil
+
+# =============================================================================
+#  Generates QIR files for all *.qs files in this folder
+#  Accepts arguments:
+#    path to qsc.exe (absolute or rely on Path env)
+#
+#  For example: "generate.py qsc.exe"
+# =============================================================================
+
+# =============================================================================
+def log(message):
+  now = datetime.datetime.now()
+  current_time = now.strftime("%H:%M:%S")
+  print(current_time + ": " + message)
+# =============================================================================
+
+root_dir = os.path.dirname(os.path.abspath(__file__))
+
+# parameters
+qsc = sys.argv[1] # argv[0] is the name of this script file
+
+# if no file name argument, process all qs files in this folder, otherwise treat all additional args as input files
```

Comment:
The command line handling logic here is fairly complex; it may be worth using the built-in [`argparse` module](https://docs.python.org/3/library/argparse.html) or a third-party module like `click` to help provide help to the user in running this script.

Reply:
I think this file has been consolidated [here](https://github.com/microsoft/qsharp-runtime/blob/main/src/QirRuntime/generateqir.py) and no longer receives any arguments. I'll remove this file from this change to leverage the latest changes in QIR generation.

Reply:
File removed.